package.json

```diff
@@ -0,0 +1,76 @@
+{
+  "name": "astro-typedoc",
+  "version": "0.0.0",
+  "description": "",
+  "keywords": [],
+  "author": "Felix Khafizov",
+  "license": "MIT",
+  "repository": "ai/NAME",
+  "sideEffects": false,
+  "engines": {
+    "node": "^16.0.0 || ^18.0.0 || >=20.0.0"
```

Comment:
Don’t copy boilerplate as-is. It is an issue. On CI we test only 20 and 18, but here have 16, 18, 20.

I fixed boilerplate and removed Node.js 16 from this line.